app/components/Schedule.tsx

```diff
@@ -99,8 +98,9 @@ function Talks({ talks }: TalksProps) {
                 ? "mb-2 border-l-8 pl-2 xl:m-0 xl:w-1/3"
                 : "w-full border-l-8"
             }`}
-            // @ts-expect-error
-            style={{ borderColor: Tracks[talk.Track]?.border }}
+            style={{
+              borderColor: TRACKS[talk.Track as keyof typeof TRACKS]?.border,
```

Comment:
Not a fan of this cast, I'd like to fix it in a followup. The fix basically involves typing `talk.Track` as union of string track names in the schema - this will make it completely type safe also.